content/security/for-admins/scim.md

```diff
@@ -58,3 +61,189 @@ You must make sure you have [configured SSO](single-sign-on/configure/_index.md)
 {{< /tab >}}
 {{< /tabs >}}
 
+## Enable SCIM in your IdP
+
+The user interface for your IdP may differ slightly from the following steps. You can refer to the documentation for your IdP to verify.
+
+{{< tabs >}}
+{{< tab name="Okta" >}}
+
+### Enable SCIM
+
+1. Go to the Okta admin portal.
+2. Go to the app you created when you configured your SSO connection.
+3. On the app page, go to the **General** tab and select **Edit App Settings**.
+4. Enable SCIM provisioning, then select **Save**.
+5. Now you can access the **Provisioning** tab. Navigate to this tab, then select **Edit SCIM Connection**.
+6. To configure SCIM in Okta, set up your connection like the following:
+    - SCIM Base URL: SCIM connector base URL (copied from Docker Hub)
+    - Unique identifier field for users: `email`
+    - Supported provisioning actions: **Push New Users** and **Push Profile Updates**
+    - Authentication Mode: HTTP Header
+    - SCIM Bearer Token: HTTP Header Authorization Bearer Token (copied from Docker Hub)
+7. Select **Test Connector Configuration**.
+8. Review the test results.
+9. Select **Save**.
+
+### Enable synchronization
+
+1. Go to **Provisioning > To App > Edit**.
+2. Enable **Create Users**, **Update User Attributes**, and **Deactivate Users**.
+3. Select **Save**.
+4. Remove unnecessary mappings. The necessary mappings are:
+    - Username
+    - Given name
+    - Family name
+    - Email
+
+{{< /tab >}}
+{{< tab name="Entra ID SAML 2.0" >}}
+
+1. In the Azure admin portal, go to **Enterprise Applications**, then select the **Docker** application you created when you set up your SSO connection.
+2. Go to **Provisioning** and select **Get Started**.
+3. Select **Automatic** provisioning mode.
+4. Enter the SCIM Base URL and API Token from Docker Hub into the **Admin Credentials** form.
+5. Test the connection, then select **Save**.
+6. Go to  **Mappings** , then select **Provision Azure Active Directory Groups**.
+7. Set the **Enabled** value to **No**.
```

Comment:
**thought:** maybe we should add a note that this would only be enabled for Group Mapping (or maybe that would be confusing and we could just add that in the Group Mapping section 🤔 )

Reply:
Would that be setting **Enabled** to **Yes** in the case of Group Mapping only? I see what you mean, but it might be confusing to mention Group Mapping here. I'll be sure to call this out when I update those docs, though!